### PR TITLE
feat(): rework network, fixes #1, fixes #4

### LINKED ISSUE #1: slave vulnerable to DNSSEC amplification attack
Our slave does not define any firewall rules that stop DNSSEC amplificationn attacks and leaves that task to the docker host. We should introduce a mitigation that ships with the slave image.

### LINKED ISSUE #4: ns: configure IPv6 address explicitly
advantages:

- client IPs are visible to pdns (so far, it sees the docker bridge IP)
- rate limits with ip6tables can be done in a less confusing way

### PR DESCRIPTION
**Pull request by peterthomassen**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -43,3 +50,17 @@ services:
 
 volumes:
   db_mysql:
+
+networks:
+  # Note that it is required that the front network ranks lower (in lexical order)
```
Comment (nils-wisiol) [inline docker-compose.yml]: change: "in lexical order by name"

**Review by nils-wisiol — COMMENTED** [review]
honestly I don't know enough about docker networking to make a qualified decision on this

File: docker-compose.yml (hunk 1 above)
Comment (peterthomassen) [reply docker-compose.yml]: fixed

**Comment by peterthomassen** [comment]
I addressed your docs suggestion. I did thorough testing with this and think it's sound. Ready to merge?

**Comment by nils-wisiol** [comment]
merge! :+1: :1st_place_medal:

**Pull request merged**